### PR TITLE
rebuild 0.5.1

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] checked license
* [x] bumped build number
* [x] adjusted dependency requirements

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,17 +12,18 @@ source:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:12]: v0.6.0 has been released https://github.com/arviz-devs/xarray-einstats/tree/v0.6.0

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,17 +12,18 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  number: 0
```
As it's a first build

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,17 +12,18 @@ source:
 
 build:
   noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [py<39]
```
See https://github.com/arviz-devs/xarray-einstats/blob/4d9b8c76284e4e56f1691060b200dd4b83424382/pyproject.toml#L9

File: recipe/meta.yaml [hunk 4]
```diff
@@ -12,17 +12,18 @@ source:
...
-    - pip >=21.3
-    - python >=3.8
+    - pip
+    - python
     - flit
+    - wheel
   run:
-    - numpy >=1.20
-    - python >=3.8
+    - numpy {{ numpy }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - numpy >=1.21
```
 The `{{ numpy }}` notation is for `host` not for `run`

File: recipe/meta.yaml [hunk 5]
```diff
@@ -12,17 +12,18 @@ source:
...
+    - python
     - scipy >=1.6
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - scipy >=1.7
```
For v0.6.0 https://github.com/arviz-devs/xarray-einstats/blob/4d9b8c76284e4e56f1691060b200dd4b83424382/pyproject.toml#L30

File: recipe/meta.yaml (hunk 4 above)
Comment (boldorider4) [reply recipe/meta.yaml]: how about `{{ pin_compatible('numpy') }}`? `cbc.yaml` supports only `numpy >= 1.21`

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml:12]: done.

File: recipe/meta.yaml (hunk 4 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Yes, it's possible

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:12]: It's still 0.5.1. And after updating the version, please modify a PR title

Comment (boldorider4) [reply recipe/meta.yaml:12]: going to update soon...sorry I needed to go back because of a minor issue

Comment (boldorider4) [reply recipe/meta.yaml:12]: done.

**Review by skupr-anaconda — APPROVED** [review]

**Review by lorepirri — APPROVED** [review]

**Pull request merged**